docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -10,28 +10,45 @@
 Cube regridding and interpolation
 =================================
 
-Iris builds upon interpolation schemes implemented by scipy, and other packages,
-to add powerful cube-aware regrid and interpolation functionality exposed through
-simple cube methods.
+Iris provides powerful cube-aware interpolation and regridding functionality,
+exposed through Iris cube methods. This functionality is provided by building
+upon existing interpolation schemes implemented by SciPy.
+
+In Iris we refer to the avaliable types of interpolation and regridding as
+`schemes`. The following are the interpolation schemes that are currently
+available in Iris:
+ * :class:`iris.analysis.Linear`, and
+ * :class:`Nearest-neighbour<iris.analysis.Nearest>`.
+
+The following are the regridding schemes that are currently available in Iris:
+ * :class:`iris.analysis.Linear`,
+ * :class:`Nearest-neighbour<iris.analysis.Nearest>`, and
+ * :class:`Area Weighted<iris.analysis.AreaWeighted>`.
```

Comment:
For what it's worth I'm :+1: **for** hyphenation i.e. `nearest-neighbour` and `area-weighted` ... nice spot @rhattersley